compiler/plugins/target/ROCM/Dialect/ROCM/Transforms/test/apply_builtin_ukernel_pdl_patterns.mlir

```diff
@@ -0,0 +1,26 @@
+// RUN: iree-opt --pass-pipeline='builtin.module(iree-rocm-apply-builtin-pdl-patterns{targets=gfx942 enable-tensor-ukernels=true})' \
+// RUN:   --mlir-print-local-scope --split-input-file %s | FileCheck %s
+
+#map2 = affine_map<(d0, d1, d2, d3) -> (d0, d1, d3)>
+#map3 = affine_map<(d0, d1, d2, d3) -> (d2, d3)>
+#map4 = affine_map<(d0, d1, d2, d3) -> (d0, d1, d2)>
+func.func @matmul_f8(%arg0: tensor<1x128x4096xf8E4M3FNUZ>, %arg1: tensor<256x4096xf8E4M3FNUZ>) -> tensor<1x128x256xf32> {
+  %cst = arith.constant 0.000000e+00 : f32
+  %0 = tensor.empty() : tensor<1x128x256xf32>
+  %1 = linalg.fill ins(%cst : f32) outs(%0 : tensor<1x128x256xf32>) -> tensor<1x128x256xf32>
+  %2 = linalg.generic {indexing_maps = [#map2, #map3, #map4], iterator_types = ["parallel", "parallel", "parallel", "reduction"]} ins(%arg0, %arg1 : tensor<1x128x4096xf8E4M3FNUZ>, tensor<256x4096xf8E4M3FNUZ>) outs(%1 : tensor<1x128x256xf32>) {
```

Comment:
Could we simplify this test a bit by using a named op and running the pass to generalize it? It adds a little bit of complexity but should be closer to the input IR, I think?